auth_saml/models/res_users.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+
+import logging
+# this is our very own dependency
+import lasso
+# this is an odoo8 dep so it should be present 'by default'
+import passlib
+
+import openerp
+from openerp import api
+from openerp import models
+from openerp import fields
+from openerp import SUPERUSER_ID
+from openerp.exceptions import ValidationError
+# import real addons name of base.res.res_users in order to call it without
+# user the super() call
+from openerp.addons.base.res.res_users import res_users as baseuser
+from openerp import _
+
+_logger = logging.getLogger(__name__)
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    saml_provider_id = fields.Many2one(
+        'auth.saml.provider',
+        string='SAML Provider',
+    )
+    saml_uid = fields.Char(
+        'SAML User ID',
+        help="SAML Provider user_id",
+    )
+
+    @api.one
```

Comment:
I think @api.one is deprecated

You should use @api.multi them ``self.ensure_one()``